tests/test_intercept.js

```diff
@@ -2414,6 +2414,27 @@ test('pending mocks doesn\'t include optional mocks', function(t) {
   t.end();
 });
 
+test('calling optionally(true) on a mock makes it optional', function (t) {
+  nock('http://example.com')
+    .get('/nonexistent')
+    .optionally(true)
+    .reply(200);
+
+  t.deepEqual(nock.pendingMocks(), []);
+  t.end();
+});
+
+test('calling optionally(false) on a mock doesn\'t make it optional', function(t) {
```

Comment:
Can you clarify what calling `optionally(false)` does instead of what it doesn't do? This phrasing is confusing.

Reply:
Sure!

Reply:
Done!